packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -291,11 +298,13 @@ Based on the scopes requested. Generally speaking, the scopes specified in a req
 | scope           | The scopes contained in the access token.                                               | String  |
 | refresh_token   | An opaque refresh token. This is returned if the `offline_access` scope is granted.     | String  |
 | id_token        | An [ID token](#id-token). This is returned if the `openid` scope is granted.            | String  |
+| interaction_handle | An opaque handle that can be used to start a transaction with the Okta Identity Engine. See guide for more details.            | String  |
 
 #### List of errors
 
 | Error Id                 | Details                                                                                                                                                                                                    |
 | :----------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| interaction_required     | The client is configured to use the Interaction Code Flow and user interaction is required to complete the request.                                                                                        |
```

Comment:
nit: may be we can say here that with this error there will be `interaction_handle` returned that need to be used for user interaction.

Reply:
Good point. @susanharper-okta can you help me with the phrasing? 

The client is configured to use the Interaction Code Flow and user interaction is required to complete the request. This error returns an `interaction_handle`, which can be used to...

There should be a link to wherever we describe the `interaction_handle` in detail. I'm trying to decide if we also need to give a quick explanation here of how the handle can be used (e.g. if I should finish that trailed off sentence or not).

Reply:
The client is configured to use the Interaction Code flow and user interaction is required to complete the request. This error returns an `interaction_handle` that you can use to (I would def give a brief explanation here and then link off to the interaction_handle in detail content).